v/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM thevlang/vlang
+
+WORKDIR /app
+
+{{#files}}
+  COPY '{{source}}' '{{target}}'
+{{/files}}
+
+RUN v . -o server
```

Comment:
do you not have any build action ? I mean compilation ....

Reply:
v . - is build action.. 😅
The first time I failed to set up a two-stage image. I'll try again later

Reply:
Check it out now

Reply:
> v . - is build action.. 😅

Why not `v -prod .` ?